Week_01_Basics/Assignment/portfolio.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+  <head>
+    <meta charset="UTF-8" />
+    <link href="./style.css" rel="stylesheet" type="text/css" />
+    <title>Joe Johnson: Portfolio</title>
+  </head>
+  <body>
+    <h2>EXPERIENCE</h2>		
+    <p>I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.</p>
+    <h2>RELEVANT EXPERIENCE</h2>
+    <!--RESUME PAGE-->
+    <ul>
+      <li><p>Web Developer, Relaxr 2015 - present</p>
+          <p>Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</p></li>
```

Comment:
This might be a better indentation, it shows clearly that the two paragraphs are childrens of your list item
```
<li>
    <p>Web Developer, Relaxr 2015 - present</p>
    <p>Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</p></li>
```